DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/theDocumentStore/TheDocumentStore.java

```diff
@@ -0,0 +1,46 @@
+package net.ravendb.demo.basics.theDocumentStore;
+//region Usings
+import net.ravendb.client.documents.IDocumentStore;
+import net.ravendb.client.documents.DocumentStore;
+package com.journaldev.singleton;
+//endregion
+
+//region Demo
+//region Step_1
+public class DocumentStoreHolder {
+//endregion
+    //region Step_2
+    private static IDocumentStore store;
+    //endregion
+
+    //region Step_3
+    private IDocumentStore createDocumentStore()
+    //endregion
+        //region Step_4
+        string serverURL = "http://localhost:8080";
```

Comment:
Does it even compile? string using lower case in java?

Reply:
fixed